### DISCUSSION TITLE
请问可以引用单个组件吗

### DISCUSSION THREAD

**Discussion by suedar** [DouyinFE/semi-design#2400] [Q&A]
已经引用了一个组件库了，因为不能满足需求想要引入semi的其中一个组件，想请问一下semi支持单个组件引入吗

**Comment by pointhalo** [comment]
我们是整体发包的，不存在单个组件单个包的引入，单个组件单个包的设定会造成版本碎片化严重。
实际上对于你来说，都是能shaking的，从一个整包import 一个module 跟从一个单独包import一个module没区别。
